api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -824,6 +824,15 @@ private void checkGetPersonNameResultForVoidedPerson(PersonName expectedPersonAd
 		
 		assertEquals(expectedPersonAddress, actualPersonName);
 	}
+
+	@Test
+	public void shouldSetDeadToTrueIfSetDeathdate() {
+
+				Person p = new Person();
+				Date deathDate = new Date();
+				p.setDeathDate(deathDate);
+				assert (p.getDead());
```

Comment:
Did you intentionally decided not to use "Assert.assertTrue()" as provided in the ticket description?